lib/main_common.pm

```diff
@@ -534,10 +545,17 @@ sub load_system_role_tests {
         loadtest "installation/logpackages";
     }
     loadtest "installation/disable_online_repos" if get_var('DISABLE_ONLINE_REPOS') && !get_var('OFFLINE_SUT');
-    loadtest "installation/installer_desktopselection" if is_opensuse;
-    loadtest "installation/system_role" if is_caasp('kubic');
+    # SYSTEM_ROLE_STYLE will be included in Mediums for openSUSE products progressively offering this new dialog
+    # i.e.: not all staging will receive it at the same time
+    if (is_opensuse) {
+        if (get_var('SYSTEM_ROLE_STYLE') || is_caasp('kubic')) {
```

Comment:
I'm don't think that is_opensuse returns true in case of caasp.
So we can write something like

```perl
if (get_var('SYSTEM_ROLE_STYLE') || is_caasp('kubic')) {
 loadtest "installation/system_role";
} elsif (is_opensuse) {
 loadtest "installation/installer_desktopselection";
}
```